setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
```

Comment:
I don't see why it shouldn't be?

`cmsgemos` is required since `ana_scans.py` makes use of `runCommand`, and `envCheck` from `gempython` inside of cmsgemos